history/taskana-simplehistory-rest-spring/src/test/java/pro/taskana/rest/ExampleDocumentationApplication.java

```diff
@@ -21,7 +21,7 @@
 
 /** Example Application to create the documentation. */
 @SpringBootApplication
-@ComponentScan(basePackages = "pro.taskana.rest.simplehistory")
+@ComponentScan("pro.taskana.rest.simplehistory")
```

Comment:
Should be deletable due to the class residing in the above pro.taskana.rest package

Reply:
done